packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/modeledDataStreamTable/modeledDataStreamTable.tsx

```diff
@@ -104,22 +110,29 @@ export function ModeledDataStreamTable({
         />
       }
       footer={
-        <Box float='right'>
-          <SpaceBetween direction='horizontal' size='xs'>
-            <Button disabled={collectionProps.selectedItems?.length === 0} onClick={() => actions.setSelectedItems([])}>
-              Reset
-            </Button>
-            <Button
-              variant='primary'
-              disabled={collectionProps.selectedItems?.length === 0 || selectedWidgets.length !== 1}
-              onClick={() => {
-                onClickAddModeledDataStreams(collectionProps.selectedItems as unknown as ModeledDataStream[]);
-              }}
-            >
-              Add
-            </Button>
-          </SpaceBetween>
-        </Box>
+        <div ref={componentRef}>
```

Comment:
this and unmodeled component footer is the same ? worth combining them ?

Reply:
done.